Validation/Geometry/test/runP_Tracker_cfg.py

```diff
@@ -62,9 +62,9 @@
     )
 
 if options.label not in _ALLOWED_LABELS:
-    print "\n*** Error, '%s' not registered as a valid components to monitor." % options.label
-    print "Allowed components:", _ALLOWED_LABELS
-    print
+    print("\n*** Error, '%s' not registered as a valid components to monitor." % options.label)
+    print("Allowed components:", _ALLOWED_LABELS)
+    print()
```

Comment:
This `print` is not needed

Reply:
I would say it can be fixed in another round

Reply:
this PR doesn't add the printout - it preserves current behavior.

Reply:
@davidlange6 yes, I agree, if the maintainers want to modify it it can be done in another round, no point in stopping the PR already approved for this